<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGCreateProjectDialog</class>
 <widget class="QDialog" name="GGCreateProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtTitle"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Project directory</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QLineEdit" name="txtPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Project file type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbFileType">
     <item>
      <property name="text">
       <string>Single XML</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Binary</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Project file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtProjectFile">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Initial Scene name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtDefaultSceneName">
     <property name="text">
      <string>Intro</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Initial Scene media directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtDefaultSceneDir">
     <property name="text">
      <string>intro</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>43</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtTitle</tabstop>
  <tabstop>txtPath</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>cmbFileType</tabstop>
  <tabstop>txtProjectFile</tabstop>
  <tabstop>txtDefaultSceneName</tabstop>
  <tabstop>txtDefaultSceneDir</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPath</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>checkOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtTitle</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>checkOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtDefaultSceneName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>checkOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtDefaultSceneDir</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>checkOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPath</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>updateFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPath</sender>
   <signal>editingFinished()</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>updateFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbFileType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>updateFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GGCreateProjectDialog</receiver>
   <slot>updateFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkOk()</slot>
  <slot>updateFileName()</slot>
 </slots>
</ui>
